cdp_scrapers/instances/houston.py

```diff
@@ -0,0 +1,531 @@
+                                            one_matter_type = matter_type.text.split(
+                                                "-"
+                                            )[0].strip()
+                                            break
+                                    event_minutes_items.append(
+                                        ingestion_models.EventMinutesItem(
+                                            minutes_item=ingestion_models.MinutesItem(
+                                                get_matter_name(link)
+                                            ),
+                                            matter=ingestion_models.Matter(
+                                                name=get_matter_name(link),
+                                                matter_type=one_matter_type,
+                                                title=get_matter_title(link),
+                                            ),
+                                        )
+                                    )
+                else:
+                    continue
+                break
+        else:
+            continue
+        break
+
+    # get minutes after matter
+    allTable = remove_extra_type(event).find_all("table")[1].find_all("table")
+    for table in allTable:
+        for td in table.find_all("td", id="column2"):
+            if "END OF CONSENT AGENDA" in td.text:
+                all_afterm_minutes = table_link.find_all_next("a", href=True)
+                for minutes in all_afterm_minutes:
+                    if minutes is not None and minutes.text != "VIDEO":
+                        minutes_link = (
+                            "https://houston.novusagenda.com/agendapublic//"
+                            + minutes["href"]
+                        )
+                        minute_types = minutes.find_all_previous(
+                            "td", id="column2", class_="style1"
+                        )
+                        one_minute_type = ""
+                        for minute_type in minute_types:
+                            if minute_type is not None and minute_type.text != "":
+                                one_minute_type = minute_type.text.split("-")[0].strip()
+                                break
+                        if "MATTERS HELD" not in one_minute_type:
+                            event_minutes_items.append(
+                                ingestion_models.EventMinutesItem(
+                                    minutes_item=ingestion_models.MinutesItem(
+                                        get_matter_name(minutes_link)
+                                    )
+                                )
+                            )
+    print("end get items")
+    return event_minutes_items
+
+
+# Big Functions
+main_URL = "https://houstontx.new.swagit.com/views/408"
+main_page = requests.get(main_URL)
+main = BeautifulSoup(main_page.content, "html.parser")
```

Comment:
Place these function calls inside the `get_events` function. Don't call / process anything outside of functions!